### ISSUE TITLE
The Ignorable attribute contains an undefined prefix

### ISSUE THREAD

**Issue by JanVanderschoot** [OfficeDev/Open-Xml-PowerTools#48]
When the example document NumberedListTest.docx is passed through SimplifyMarkup, the resulting docx lists w15 in the Ignorable attribute in word/document.xml, but it misses a namespace declaration for w15.
If the resulting document is subsequently passed through Expand-DocxFormatting.ps1the same problem occurs both in word/document.xml and word/styles.xml.
These problems are signaled by both the Word executable and Get-OpenXmlValidationErrors.ps1.

**Comment by EricWhiteDev** [comment]
@JanVanderschoot You are right, MarkupSimplifier should do the right thing in this scenario, and certainly should not generate an invalid document, or a document that will not open.  I'll take a look.

**Comment by EricWhiteDev** [comment]
@JanVanderschoot I have decided to add new XUnit tests for MarkupSimplifier, which go through all sample DOCX files in TestFiles, (will add the above file to TestFiles as well), simplify markup, and then validate, failing if MarkupSimplifier produces invalid markup.  I think there will be a couple of places where we may run into issues where MarkupSimplifier needs to be fixed.

I will be doing some more Open-Xml-PowerTools development in a couple of months.  Leaving this issue open, and will address it in a methodical way at that point.

**Comment by cpmcgrath** [comment]
I think this is the problem I'm experiencing. I run the MarkupSimplifier.SimplifyMarkup, and when I try
to open the resulting document I get an error, "The XML data is invalid according to the schema. Location   Part: /word/styles.xml, Line: 0, Column: 0"

This always happens for me. Even with a brand new document with nothing in it.
Using Word 2016 32bit (16.0.6769.2017)

**Comment by cpmcgrath** [comment]
I created a new blank document in Word 2016 32bit, Saved it and made a copy of it.
On both I opened them with the WordprocessingDocument, with one I did a simplify, and then disposed them. Attached are the results. The With Simplify causes the above error when you open it with Word.

[result-NoSimplify.docx](https://github.com/OfficeDev/Open-Xml-PowerTools/files/313220/result-NoSimplify.docx)
[result-WithSimplify.docx](https://github.com/OfficeDev/Open-Xml-PowerTools/files/313221/result-WithSimplify.docx)
